src/Amadeus/Client/Struct/Service/StandaloneCatalogue/PassengerInfoGroup.php

```diff
@@ -0,0 +1,60 @@
+<?php
```

Comment:
Same remark.

Reply:
We need index of each passenger instead of group of passenger that is used in other request. For this we need this Class file.